pages/etherpad/[[...roomId]].js

```diff
@@ -266,6 +288,7 @@ export default function Etherpad() {
                     <>
                         <ServiceSubmenu
                             title={<h2>{ etherpadPath }</h2>}
+                            disabled={!auth.connectionStatus.etherpad}
```

Comment:
Why is this still needed? If `!auth.connectionStatus.etherpad` is true (so when the MyPads server is down) we will never ever show this submenu anymore anyways, right?!